### PR TITLE
System.cpp: Do not create VFS directories outside of RPCS3 dir

### PR DESCRIPTION
**Pull request by elad335**
If the user made a mistake and removed a custom Virtual File System directory or moved it, RPCS3 shouldn't create the old path by force and use it. Instead encourage him to fix it himself, it also avoids his confusion as to why games won't appear on the new path.

### CODE REVIEW

File: rpcs3/Emu/System.cpp [hunk 1]
```diff
@@ -368,9 +368,22 @@ void Emulator::Init(bool add_only)
 	// Disable incompatible settings
 	fixup_ppu_settings();
 
+	bool must_create_inside_emu_dir = false;
```
Comment (Megamouse) [inline rpcs3/Emu/System.cpp:371]: please just use a second param. this is very confusing

Comment (elad335) [reply rpcs3/Emu/System.cpp]: Fixed.

File: rpcs3/Emu/System.cpp [hunk 2]
```diff
@@ -385,55 +396,61 @@ void Emulator::Init(bool add_only)
...
+		{
+			make_path_verbose(dev_hdd0 + "game/", true);
+			make_path_verbose(dev_hdd0 + reinterpret_cast<const char*>(u8"game/＄locks/"), true);
+			make_path_verbose(dev_hdd0 + "home/", true);
+			make_path_verbose(dev_hdd0 + "home/" + m_usr + "/", true);
+			make_path_verbose(dev_hdd0 + "home/" + m_usr + "/exdata/", true);
+			make_path_verbose(save_path, true);
+			make_path_verbose(dev_hdd0 + "home/" + m_usr + "/trophy/", true);
+
+			if (!fs::write_file(user_path, fs::create + fs::excl + fs::write, "User"s, true))
```
Comment (Megamouse) [inline rpcs3/Emu/System.cpp:420]: I think you added a true here by mistake

Comment (elad335) [reply rpcs3/Emu/System.cpp]: Fixed,

**Review by Megamouse — APPROVED** [review]

**Pull request merged**